ansible/main.yml

```diff
@@ -48,6 +48,20 @@
   roles:
     - { role: okfn-org-manager, tags: ['heroku-backup'] }
 
+- hosts: s138.okserver.org
+  sudo: true 
```

Comment:
Shall we just add the following for roles that need `p.yml`? That'll make it easier for us.

```
vars:
  - private_dir: ../../credentials/ansible/